Sources/Plasma/Apps/plUruLauncher/plClientLauncher.cpp

```diff
@@ -389,7 +389,7 @@ void plClientLauncher::InitializeNetCore()
     uint32_t num = GetGateKeeperSrvHostnames(addrs);
 
     NetCliGateKeeperStartConnect(addrs, num);
-    NetCliGateKeeperFileSrvIpAddressRequest(true, [this](auto result, auto addr) {
+    NetCliGateKeeperFileSrvIpAddressRequest(true, [this](auto result, const auto& addr) {
```

Comment:
I hate that `auto` doesn't do this by default, even though the callback type defines this parameter as `const ST::string&`...